tests/clone_test.go

```diff
@@ -869,3 +849,23 @@ func stopCloneVM(virtClient kubecli.KubevirtClient, vm *virtv1.VirtualMachine) (
 
 	return virtClient.VirtualMachine(vm.Namespace).Patch(context.Background(), vm.Name, types.JSONPatchType, patch, v1.PatchOptions{})
 }
+
+func withFirmware(firmware *virtv1.Firmware) libvmi.Option {
+	return func(vmi *virtv1.VirtualMachineInstance) {
+		vmi.Spec.Domain.Firmware = firmware
+	}
+}
+
+func createSourceVM(options ...libvmi.Option) *virtv1.VirtualMachine {
+	vmi := libvmifact.NewCirros(options...)
```

Comment:
Can we add the default options to here and append them unconditionally, avoiding the need to append them in every test?

Reply:
thing is - that default Option (some labels and annotations) are test specific. 
Defining them outside of the test scope seems off to me, so I thought it would be better to append them every time. 
The refactored defaultOptions is global on the test's scope, and appended on each test - but at least it's not global in the scope of the file/folder. I think it's the lesser of evils..

Reply:
> thing is - that default Option (some labels and annotations) are test specific.

You're generally right, but what's bothering me is that now it seems that every test chooses to include these options explicitly, as if they're important for every test. I say we either need to use these options only for the tests necessary, or for all tests. This middle way is the most confusing to me.

Reply:
> hmm it was my impression that they are important for every test, as there is an explicit expectEqualAnnotations/Labels that checks that make sure both source and target annotation/labels stay the same.

From a quick glance it seems that `expectEqualAnnotations/Labels` is not used by all tests, although I'm not sure there's a sense behind that.

> Well if they are not needed on every test, why are they default? Should they be relevant only to specific tests?

Not sure why they're default, but my point is that we either always need to test it or make sure that it is tested in the right tests. Currently the PR gives a false sense that the tests that are using it truly need it and this fact needs to be revisited. In any case this shouldn't block this PR.